.github/workflows/tests.yml

```diff
@@ -0,0 +1,47 @@
+name: Tests
+
+on:
```

Comment:
I wonder if just `on: [push]` wouldn't be the same... 🤔

Reply:
Don't know, to be honest, lets try: 567b5f5 :)